tools/compile.lua

```diff
@@ -0,0 +1,8 @@
+for line in io.lines() do
```

Comment:
Write it in awk or another shell script. Relying on Lua seems unnecessary.

Reply:
This script is not really required, but it is added to handle some differences in output format.

In the old implementation, `printf("%s\n", "hello");` will output

```text
Script log: hello
```

After recompiling with musl, it will output

```text
Script log: hello
Script log:
```

The difference is that the old implementation we rewrote `printf`, while musl only rewrote `writev`.

To handle the differences here, I wrote this script. At the same time, I updated ckb-debugger: https://github.com/nervosnetwork/ckb-standalone-debugger/commit/af0969102a84757a8695631402f05d42a419811f

This script ensures that the same results are always obtained when using different versions of ckb-debugger.

Reply:
The reason for using lua is just because this project already has other lua scripts